<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="eh_layout">
     <item>
      <widget class="QCheckBox" name="eh_signal_enabled">
       <property name="text">
        <string>Every hour signal</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="eh_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="eh_config_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="qh_layout">
     <item>
      <widget class="QCheckBox" name="qh_signal_enabled">
       <property name="text">
        <string>Quarter hour signal</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="qh_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="qh_config_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="custom_layout">
     <item>
      <widget class="QCheckBox" name="custom_enabled">
       <property name="text">
        <string>Custom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="custom_period">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="custom_offset_label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>offset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="custom_offset">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="custom_config_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eh_signal_enabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>eh_config_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qh_signal_enabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>qh_config_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_enabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>custom_period</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_enabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>custom_offset_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_enabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>custom_offset</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_enabled</sender>
   <signal>toggled(bool)</signal>
   <receiver>custom_config_btn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
